cmd/eskip/media_defaults.go

```diff
@@ -0,0 +1,45 @@
+package main
```

Comment:
pls, call it 'mediadefaults.go'. the _ is used to separate tests, platform, architecture and similar things.